pkg/rhoda/rhoda_test.go

```diff
@@ -18,165 +22,379 @@ import (
-				//}
-				//fmt.Print(string(inventoryData))
-				//	Eventually(isInventoryReady(c, inventory.ObjectMeta.Name, namespace), 30, 5).Should(BeTrue())
 			})
-		})
+		}
+	})
+
+	It("Console Login UI and pages access check", func() {
+		fmt.Println("Login")
+		ctx, cancel := chromedp.NewContext(context.Background())
+		defer cancel()
+		//Setting timeout
+		//ctx, cancel = context.WithTimeout(ctx, 5*time.Second)
+		//defer cancel()
+
+		var nodes []*cdp.Node
+		//selector for checking the Data Services button
+		selector := "#page-sidebar div ul li button"
+		url := "https://console-openshift-console.apps.rhoda-lab.51ty.p1.openshiftapps.com/dashboards"
+		// url:= "https://console-openshift-console.apps.rhoda-sp-prod.lue0.p1.openshiftapps.com/dashboards"
+		var data string
+
+		if err := chromedp.Run(ctx,
+			SetCookie("openshift-session-token", config.BearerToken, "console-openshift-console.apps.rhoda-lab.51ty.p1.openshiftapps.com", "/", false, false),
+			chromedp.Navigate(url),
+			chromedp.WaitVisible(`#page-sidebar`),
+			chromedp.OuterHTML("html", &data, chromedp.ByQuery),
+			//	chromedp.Nodes(`button`, &nodes, chromedp.ByQueryAll),
+
+			chromedp.Nodes(selector, &nodes),
+			//chromedp.Text(`button`, &selector, chromedp.NodeVisible, chromedp.ByQuery),
+		); err != nil {
+			//panic(err)
+			AbortSuite(err.Error())
+		}
+
+		for _, node := range nodes {
+			//NodeName is the button here, looping through buttons to get Data services
+			fmt.Println(node.NodeName)
+
+			for _, child := range node.Children {
+				//getting Data Services Button
+				fmt.Println(child.NodeValue)
+				if child.NodeValue == "Data Services" {
+					//get the parent's parent which is Li to click on the Database Access button
+					li := child.Parent.Parent
+					//fmt.Println(li)
+					//checkAdminDashboard(li, ctx)
+					var data string
+					selector := "section ul li a"
+					var result []*cdp.Node
+
+					err := chromedp.Run(ctx,
+						chromedp.Nodes(selector, &result, chromedp.ByQuery, chromedp.FromNode(li)),
+					)
+					Expect(err).NotTo(HaveOccurred())
+
+					//fmt.Println(result)
+					for _, aNode := range result {
+						//temp stuff for visibility
+						text := aNode.Children[0].NodeValue
+						fmt.Println(text)
+						//	u := aNode.AttributeValue("href")
+						//	fmt.Printf("node: %s | href = %s\n", aNode.LocalName, u)
+						var textExists bool
+
+						if aNode.Children[0].NodeValue == "Database Access" {
+							u := "https://console-openshift-console.apps.rhoda-lab.51ty.p1.openshiftapps.com" + aNode.AttributeValue("href")
+							fmt.Printf("node: %s | href = %s\n", aNode.LocalName, u)
+							_, err := chromedp.RunResponse(ctx,
+								chromedp.Navigate(u),
+								chromedp.WaitVisible(`#content-scrollable button`),
+								chromedp.OuterHTML("html", &data, chromedp.ByQuery),
+								chromedp.EvaluateAsDevTools(`document.querySelector("#content-scrollable h1 div" ).innerHTML.includes("Database Access")`, &textExists),
```

Comment:
This is a text search. Would be much preferable to get the right element as done for the previous page.

Additionally, if not there, you should get the error message and log it